src/emmy/tape.cljc

```diff
@@ -142,7 +140,12 @@
   ;; This implementation is called if a tape ever makes it out of
   ;; forward-mode-differentiated function. If this happens, a [[TapeCell]]
```

Comment:
TODO update the docs here. We need a better narrative now that hints at reverse-mode in the `differential.cljc` exposition.